libraries/plugins/peerplays_sidechain/peerplays_sidechain_plugin.cpp

```diff
@@ -48,8 +52,22 @@ void peerplays_sidechain_plugin::plugin_set_program_options(
    )
 {
    cli.add_options()
-         ("parameter", boost::program_options::value<uint32_t>(), "Parameter")
-         ("optional-parameter", boost::program_options::value<uint32_t>(), "Optional parameter")
+         //("bitcoin-node-ip", bpo::value<string>()->implicit_value("127.0.0.1"), "IP address of Bitcoin node")
+         //("bitcoin-node-zmq-port", bpo::value<uint32_t>()->implicit_value(28332), "ZMQ port of Bitcoin node")
+         //("bitcoin-node-rpc-port", bpo::value<uint32_t>()->implicit_value(18332), "RPC port of Bitcoin node")
+         //("bitcoin-node-rpc-user", bpo::value<string>(), "Bitcoin RPC user")
+         //("bitcoin-node-rpc-password", bpo::value<string>(), "Bitcoin RPC password")
+         //("bitcoin-address", bpo::value<string>(), "Bitcoin address")
+         //("bitcoin-public-key", bpo::value<string>(), "Bitcoin public key")
+         //("bitcoin-private-key", bpo::value<string>(), "Bitcoin private key")
```

Comment:
remove above.

Reply:
I want to keep this for now, since development version contains parameters for our test environment. I will remove this once we are nearly completed, and I wrote instructions on how to configure SON properly.